### DISCUSSION TITLE
Better checklists

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#536] [Completed things] [answered]
Currently the checklists that are provided, are very basic and do not cover the full operations of the aircraft, but it wouldn’t take long to add some more stuff. The complexity of the zibo’s simple checklists on their EFB would be perfect. Include the full procedures, for example not just landing, but so also checklists for ILS landing, auto land, visual, etc. Non-normal situations would be pretty cool also, but not required. I would imagine this not taking too long so could be done much quicker than other things. Thanks.

**Comment by aeroniemi** [comment, marked as answer]
It's possible for users to make their own checklists (indeed theoretically on a per-livery basis), so if you'd like to make your own, you can!

**Reply by Simleybob** [reply]
Okay, it would be better tho if it came with the plane, I honestly doubt people would do that.
